src/registrar/templates/domain_detail.html

```diff
@@ -28,8 +28,8 @@
     <br>
 
     {% url 'domain-dns-nameservers' pk=domain.id as url %}
-    {% if domain.nameservers|length > 0 %}
-      {% include "includes/summary_item.html" with title='DNS name servers' value=domain.nameservers list='true' edit_link=url %}
+    {% if domain.format_nameservers|length > 0 %}
```

Comment:
why are we changing it here and not the summary item element? there is a todo in that with this ticket # that says this is where the ticket needs to be fixed. @therealslimhsiehdy do you remember us adding that todo when we did nameservers, or was this intentional?

Reply:
GREAT catch, I saw it in the nameserver function, but completely forgot about it in the `includes/summary_item.html` file. Will remedy this!